README.md

```diff
@@ -8,78 +8,39 @@
 <img src="tekton-triggers.png" alt="Tekton Triggers logo (Tekton cat playing with a ball)"></img>
 </p>
 
-Triggers is a Kubernetes
+Tekton Triggers is a Kubernetes
 [Custom Resource Definition](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/)
-(CRD) controller that allows you to extract information from events payloads (a
-"trigger") to create Kubernetes resources. If you are looking to start with triggers
-please take a look at the [getting started guide](./docs/getting-started/README.md). 
+(CRD) controller that allows you to create Kubernetes resources based on information it extracts from event payloads called **triggers**.
```

Comment:
This makes it sound like the event payloads is called triggers which is confusing.
Maybe we could just remove "called triggers" from this sentence?

Reply:
Sure, it sounded like the original version calls them that, so maybe I misunderstood. Removed.